engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterLaunchEngine.m

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterLaunchEngine.h"
+
+@interface FlutterLaunchEngine () {
+  BOOL _didGrabEngine;
+  FlutterEngine* _engine;
+}
+@end
+
+@implementation FlutterLaunchEngine
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    self->_didGrabEngine = NO;
+  }
+  return self;
+}
+
+- (FlutterEngine*)engine {
```

Comment:
nit: `createAndRunEngineIfNeeded`

Reply:
This getter is automatically created by the property and has to match.